app/services/WebPlayer.java

```diff
@@ -0,0 +1,26 @@
+package services;
+
+import ttt.Player;
+import ttt.game.Board;
+import ttt.game.Marks;
+
+public class WebPlayer implements Player {
+
+    private Marks mark;
+
+    public WebPlayer(Marks mark) {
+        this.mark = mark;
+    }
+
+    public Marks getMark() {
+        return mark;
+    }
+
+    public Integer getLocation(Board board) throws Exception {
+        return null;
+    }
+
+    public Class playerType() {
+        return getClass();
+    }
+}
```

Comment:
Also as you mentioned, with the web game, often input validation completely goes away, as the web interface only presents 'valid' moves, so no point in having an Input if you don't need one.